### PR TITLE
Adapt tabbed instructions & other fixes for nl-NL localization

### PR DESCRIPTION
**Pull request by Joery**
- Add tabbed instructions to seperate Windows/macOS
- Replace png with svg for dark theme
- Rephrased some sentences
- Small translation improvements (mix of formal / informal to all informal)

In sync with English site as of commit f4f8afbeb929221e077341074664575e478bcd17

### CODE REVIEW

**Comment by Joery** [comment]
Ready to merge

File: src/nl/help/guides/getting-started.md [hunk 1]
```diff
@@ -8,89 +8,98 @@ lang: nl-NL
 ## Installatie
 **Paperback** is momenteel _niet_ op **TestFlight** en _vereist_ het gebruik van **AltStore** om op je iPhone of iPad te installeren. In de toekomst zal TestFlight worden gebruikt wanneer de hoofdontwikkelaar een [Apple Developer Account](https://developer.apple.com/programs/) heeft.
 
-::: tip Video tutorial
-Als je liever een video-tutorial gebruikt, heeft Artesians er een mooie gemaakt. Je vindt deze [hier](https://www.youtube.com/watch?v=n1KRwsxNiWY).
+::: tip Video-tutorial
+Als je liever een video-tutorial gebruikt, heeft Artesians er een gemaakt. Je vindt deze [hier](https://www.youtube.com/watch?v=n1KRwsxNiWY).
```
Comment (FramboisePi) [inline src/nl/help/guides/getting-started.md:12]: You should add macOS tutorial: https://www.youtube.com/watch?v=CjPjsF4yJ0M

Comment (Joery) [reply src/nl/help/guides/getting-started.md]: Added a link for macOS, thanks Framboise

File: src/nl/help/faq/README.md [hunk 2]
```diff
@@ -15,81 +15,83 @@ Er zijn geen fatsoenlijke manga-apps op **iOS** die _niet-opdringerige_ adverten
...
 ### Het sorteren op nieuwste updates werkt niet
-Het werkt, maar het respecteert de geselecteerde taal van de gebruiker niet, probeer in plaats daarvan te sorteren op Update Count.
+Dit werkt wel, maar het houd geen rekening met de geselecteerde taal van de gebruiker. Probeer in plaats daarvan te sorteren op **Update Count**.
+
+---
 
 ### Ik heb een probleem dat hier niet wordt vermeld
-Als het een bug is, maak dan een bugrapport aan op GitHub (In het Engels). Vraag anders om hulp in het Discord #support-kanaal.
\ No newline at end of file
+Als het een bug is, maak dan een bugrapport aan op (GitHub)[https://github.com/Paperback-iOS/app/issues] (In het Engels). Vraag anders om hulp in het Discord #support-kanaal (In het Engels).
```
Comment (FramboisePi) [inline src/nl/help/faq/README.md:97]: The link wont work. Use:
```markdown
[GitHub](https://github.com/Paperback-iOS/app/issues)
```

File: src/nl/help/faq/README.md [hunk 3]
```diff
@@ -15,81 +15,83 @@ Er zijn geen fatsoenlijke manga-apps op **iOS** die _niet-opdringerige_ adverten
...
 
 ### App loopt vast bij het opstarten
-**Publieke build** gebruikers moeten controleren of ze iOS 13.4+ gebruiken. **Beta / Patron build** gebruikers moeten Paper op Discord pingen in het #supporter-kanaal (In het Engels).
+Als je de **Publieke build** gebruikt, controleer dan of je iOS 13.4+ gebruikt. Als je de **Beta / Patron build** gebruikt, stuur een bericht op het Discord #supporter-kanaal en ping Paper (In het Engels).
 
 ---
 
 ### App loopt vast bij het volgen van manga op de iPad
-Dit is een bekende Apple-bug die is opgelost in iPadOS 13.4, update naar iOS 13.4 of vermijd het volgen van manga in Paperback en gebruik de MangaDex-website om manga te volgen (en vervolgens te synchroniseren).
+Dit is een bekende Apple-bug die is opgelost in iOS 13.4, update naar iOS 13.4 of vermijd het volgen van manga in Paperback en gebruik de MangaDex-website om manga te volgen (en vervolgens te synchroniseren).
```
Comment (FramboisePi) [inline src/nl/help/faq/README.md:33]: Seems like an issue in the English version.
We are talking about crash on iPad so I would write:
```suggestion
Dit is een bekende Apple-bug die is opgelost in iPadOS 13.4, update naar iPadOS 13.4 of vermijd het volgen van manga in Paperback en gebruik de MangaDex-website om manga te volgen (en vervolgens te synchroniseren).
```

**Pull request merged**